packages/vuetify/src/components/VTimeline/VTimeline.sass

```diff
@@ -113,11 +94,35 @@
   @include horizontal
     flex-direction: row
     grid-row: 2
+    width: 100%
 
   @include vertical
+    height: 100%
     flex-direction: column
     grid-column: 2
 
+.v-timeline-divider__line
+  background: $timeline-divider-line-background
+  position: absolute
+
+  @include horizontal
+    height: $timeline-divider-line-thickness
+    width: calc(100% + #{$timeline-item-padding})
+    inset-inline-start: 0
```

Comment:
`inset` isn't supported before safari 14.1/chrome 87

Reply:
well that sucks. it was really nice not having to use rtl mixin

Reply:
hm, `padding-inline` seems to have the same support and we use that all over the place?

Reply:
padding-inline-start/end has better support than padding-inline